pipelines/cloudtrail/put_cloudtrail_trail_event_selector.fp

```diff
@@ -0,0 +1,48 @@
+pipeline "put_cloudtrail_trail_event_selector" {
+  title       = "Put event selectors to CloudTrail Trail"
+  description = "Enables log file validation for an AWS CloudTrail trail."
+
+  tags = {
+    type = "featured"
+  }
```

Comment:
@ParthaI is there a specific reason for adding the `featured` tag to this pipeline?

I think we can remove it.